Civi/Token/TokenCompatSubscriber.php

```diff
@@ -88,12 +88,9 @@ public function onEvaluate(TokenValueEvent $e) {
         $contact = array_merge($contact, $row->context['tmpTokenParams']);
       }
 
-      $contactArray = !is_array($contactId) ? [$contactId => $contact] : $contact;
```

Comment:
I don't see any reason for `$contactId` to arrive in array format here so this looks fine to me. Objections if any @eileenmcnaughton @totten ?